server/core/src/main/java/dev/slimevr/tracking/trackers/Tracker.kt

```diff
@@ -329,9 +331,11 @@ class Tracker @JvmOverloads constructor(
 		var rot = if (allowFiltering && filteringHandler.filteringEnabled) {
```

Comment:
```suggestion
		var rot = if (allowFiltering && filteringHandler.filteringEnabled && !isComputed) {
```

Reply:
This would block smoothing on Vives in entirety though wouldn't it? Maybe this should be a separate flag? Or it could be linked to `allowFiltering` instead perhaps.